src/Tribe/Cost_Utils.php

```diff
@@ -273,11 +273,11 @@ public function parse_cost_range( $costs, $max_decimals = null ) {
 			return array();
 		}
 
-		// remove any empty prices
-		$costs = array_filter( (array) $costs );
```

Comment:
This tosses out events with costs of `0`, which we need to retain. Hence the reason this is tossed out.